core/src/test/java/org/pentaho/platform/dataaccess/datasource/wizard/DatabaseConnectionConverterTest.java

```diff
@@ -33,7 +34,7 @@
 
 import com.google.gwtmockito.GwtMockitoTestRunner;
 
-@RunWith( GwtMockitoTestRunner.class )
+@Ignore @RunWith( GwtMockitoTestRunner.class )
```

Comment:
Are we ignoring this whole test?

Reply:
This file only has one test in it, and that test doesn't have any asserts or mock verifications -- it just relies on it not throwing any exceptions (and the `convertToXML` method this is testing swallows exceptions anyway).

I think we'd need to rewrite this test for it to be useful, so I figured it made sense to disable it for now. I wrote a tech debt ticket to capture this and some other tests in here that I think need attention, so we can circle back to them after the JDK changes.